package.json

```diff
@@ -15,7 +15,7 @@
     "build:server": "http-server dist",
     "dev": "nx run core:dev",
     "dev:server": "http-server build",
-    "e2e": "npx playwright install && PORT=5005 MODE=REPLAY npx playwright test",
+    "e2e": "npm run build && npx playwright install && PORT=5005 MODE=REPLAY npx playwright test",
```

Comment:
@animesh1987 Could you please do another review?
I just added everything I can think of for the test framework.